aten/src/ATen/core/boxing/impl/boxing.h

```diff
@@ -223,7 +263,7 @@ struct BoxedKernelWrapper<
     torch::jit::push(stack, std::forward<OtherArgs>(otherArgs)...);
```

Comment:
Does this handle ArrayRef arguments correctly or do they somehow not happen in calls here?

Reply:
There's definitely no logic to avoid ArrayRef args here (or in the other specializations - they all do this - are you thinking this one has a particular issue?) will follow up offline...